pkg/rbuser/generate.go

```diff
@@ -1,27 +1,14 @@
-package main
+package rbuser
 
 import (
-	"fmt"
-	"os"
 	"strconv"
-	"strings"
 
 	ldap "gopkg.in/ldap.v2"
 )
 
 // Generate user vhost conf from ldap
-func Generate(conf LdapConf, output string) (int, error) {
-	l, err := ldap.Dial("tcp", fmt.Sprintf("%s:%d", conf.Host, conf.Port))
-	if err != nil {
-		return 0, err
-	}
-	defer l.Close()
-
-	err = l.Bind(conf.User, conf.Password)
-	if err != nil {
-		return 0, err
-	}
-
+func Generate(l *ldap.Conn, output string) ([]string, error) {
```

Comment:
I feel like this is a weird to pass `ldap.Conn`. I feel like LDAP connection should be in its own structure, with GenerateVHost function.

Reply:
a struct just for passing ldap.Conn?